docs/HyperIndex/supported-networks/index.md

```
@@ -21,4 +21,13 @@ If the network that you want to index is not supported on HyperSync, please navi
 
 You can also request a network to be added to HyperSync in the [Discord](https://discord.gg/envio).
 
+### Networks available on request
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)